models/_sector/dex/trades/dex_base_trades.sql

```diff
@@ -22,6 +22,7 @@
     , ref('dex_optimism_base_trades')
     , ref('dex_polygon_base_trades')
     , ref('dex_zksync_base_trades')
+    , ref('dex_superchain_base_base_trades')
```

Comment:
By the way what is the difference between `superchain_base` and `base`?
Do you think we can put these trades under `dex_base_base_trades`?

Reply:
Hello @Hosuke,

Putting these trades under dex_base_base_trades looks much better. I have made the necessary changes; please check and let me know if anything else is required.